Makefile

```diff
@@ -105,9 +105,10 @@ run-web-app:
 test:
 	PYTHONPATH=. pytest test_${PIPELINE_PACKAGE} --cov=${PACKAGE_NAME} --cov-report term-missing
 
+# Setting a low bar here - need more tests!
 .PHONY: check-coverage
 check-coverage:
-	coverage report --fail-under=90
+	coverage report --fail-under=60
```

Comment:
this is good for now, but also worth creating an issue to increase the coverage.

i think that's pretty easily achievable by posting multiple files at once in a single request and also testing some non-200 status code cases.

Reply:
Makes sense! Looks like we can use #46 to track this.